### ISSUE TITLE
Syntax error

### ISSUE THREAD

**Issue by splincode** [Dogfalo/materialize#4145]
### Description
If you open page http://materializecss.com/dropdown.html and copy code "jQuery Plugin Initialization" and past console browser

```javascript
  $('.dropdown-button').dropdown({
      inDuration: 300,
      outDuration: 225,
      constrainWidth: false, // Does not change width of dropdown to that of the activator
      hover: true, // Activate on hover
      gutter: 0, // Spacing from edge
      belowOrigin: false, // Displays dropdown below the button
      alignment: 'left' // Displays dropdown with edge aligned to the left of button
      stopPropagation: false // Stops event propagation
    }
  );
```

I have syntax error in line " alignment: 'left' "

### Screenshot
![1485509501069](https://cloud.githubusercontent.com/assets/12021443/22366219/945d1298-e48c-11e6-8c75-7c348d52fc36.jpg)

**Comment by Nohinn** [comment]
Missing a comma after ´'left'´. You should make a pull request fixing this issue

**Comment by Dogfalo** [comment]
Fixed in 453efd027bcbe42909da7ba650f6c4402714f5c5
